app/views/pages/about.html.erb

```diff
@@ -2,68 +2,95 @@
-      fraud and reduce waste.</p>
-
-    <p>By taking ethical, open, innovative and transparent approaches to data we can build greater levels of trust with
-      citizens, whilst delivering more cost-effective and better targeted and tailored services to their needs.</p>
-
-    <p>Data is at the heart of digital transformation and a part of the
-      <a href="https://www.gov.uk/government/publications/government-transformation-strategy-2017-to-2020/government-transformation-strategy-better-use-of-data">Government
-        Transformation Strategy</a>.</p>
-
-    <p>GDS works with other organisations both within and outside government on shaping the strategic direction on how
-      data is managed, accessed and used. We work collaboratively on a range of issues with the Office for National
-      Statistics, Department for Digital, Culture, Media & Sport and The National Cyber Security Centre.</p>
-
-    <h2 class="heading-medium">Reuse information published on Find open data</h2>
-
-    <p>You can republish any of the content on Find open data, as long as you meet the conditions of either the
-      <a href="http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/">Open Government Licence</a> or
-      the licence it's covered by. For questions about a dataset, you can contact the publisher directly if they’ve
-      provided contact details on the relevant dataset page.</p>
-
-     <h2 class="heading-medium">Build services using registers</h2>
-
-    <p>You can find <a href="https://registers.cloudapps.digital/">GOV.UK registers</a> that are managed and approved by government and ready to use in your service.</p>
-    <p>Using an approved register means you can trust the data. You can also combine data from different registers using the open API. </p>
-    <p>Find out how government organisations including the Cabinet Office and Environment Agency are <a href="https://registers.cloudapps.digital/services-using-registers">using registers</a> in their services. For example, the UK Government and Parliament petitions service uses the country register to help users to choose their country.</p>
-
-     <h2 class="heading-medium">Linking to datasets on GOV.UK</h2>
-
-    <p>Data is integral to our work to transform government. Organisations can publish data on Find open data by linking to
-      datasets published on GOV.UK. To help organisations prepare and publish data on GOV.UK, we have some
-      recommendations for publishing:</p>
-
-    <ul class="list list-bullet">
-      <li><a href="https://www.gov.uk/government/collections/how-to-publish-central-government-transparency-data">central
-        government transparency data</a></li>
-      <li><a href="https://www.gov.uk/guidance/how-to-publish-prompt-payment-data">payment data</a></li>
-      <li><a href="https://www.gov.uk/guidance/how-to-publish-spend-control-data">spend control data</a></li>
-    </ul>
-
+<main role="main" id="main-content" class="govuk-main-wrapper">
```

Comment:
Just to check - will changing the id from `content` to `main-content` affect the skip to content link?

Reply:
Currently the skip link is looking for `main-content` so this actually fixes a service-wide bug.